packages/suite-web-landing/utils/switchFavicon.ts

```diff
@@ -0,0 +1,22 @@
+import { resolveStaticPath } from '@suite-utils/build';
+// switch favicon if dark mode
+export const switchFavicon = () => {
+    const icon = document.querySelector('link[rel=icon]');
+    const appleIcon = document.querySelector('link[rel=apple-touch-icon]');
+    const switchIcons = (isDarkMode: boolean) => {
+        if (isDarkMode) {
+            icon?.setAttribute('href', resolveStaticPath('/images/icons/favicon/favicon_dm.png'));
```

Comment:
![Screenshot 2021-11-04 at 14 55 07](https://user-images.githubusercontent.com/33235762/140326350-580eabcc-c015-4b89-92e6-3886ea20f4e5.png)
prefix does not work here

Reply:
aae5d5e